test/TEST-01-BASIC/test.sh

```diff
@@ -57,6 +57,13 @@ ExecStart=/bin/sh -x -c 'systemctl --failed --no-legend --no-pager > /failed ; e
 Type=oneshot
 EOF
 
+        # mask some services that we do not want to run in these tests
```

Comment:
@evverx, jup, you're right. Updated accordingly.

My tests on Semaphore succeed now, btw. So I think this is safe to merge.